policyengine_us/parameters/gov/states/oh/tax/income/credits/child/agi_partial_credit_limit.yaml

```diff
@@ -0,0 +1,10 @@
+description: Ohio Child and Dependent Care credit partial credit threshold
+values:
+  2021-01-01: 20_000
```

Comment:
This value doesn't appear on the cited page

Reply:
Sorry, it's on page 30 and I changed them in oh_senior_citizen branch